components/ai_chat/resources/untrusted_conversation_frame/components/conversation_entries/index.tsx

```diff
@@ -150,92 +147,115 @@ function ConversationEntries() {
-                            key={c.contentId}
-                            url={c.url.url}
-                            title={c.title}
-                          />)}
-                            {imageFiles.map((img) => (
-                              <AttachmentImageItem
-                                key={img.filename}
-                                uploadedImage={img}
-                              />
-                            ))}
+                          <div className={styles.humanMessageBubble}>
+                            <div className={styles.humanTextRow}>
+                              {currentEntryEdit.text}
+                              {!!entry.edits?.length && (
+                                <div className={styles.editLabel}>
+                                  <span className={styles.editLabelText}>
+                                    {getLocale(S.CHAT_UI_EDITED_LABEL)}
+                                  </span>
+                                </div>
+                              )}
+                            </div>
+                            {hasAttachments &&
+                              <div className={styles.attachmentsContainer}>
+                              {conversationContext.associatedContent.map(c => <AttachmentPageItem
+                                key={c.contentId}
+                                url={c.url.url}
+                                title={c.title}
+                              />)}
+                                {imageFiles.map((img) => (
+                                  <AttachmentImageItem
+                                    key={img.filename}
+                                    uploadedImage={img}
+                                  />
+                                ))}
+                              </div>
+                            }
                           </div>
-                        }
-                      </div>
-                    </>
-                  )}
-                  {showEditInput && (
-                    <EditInput
-                      text={latestTurnText}
-                      onSubmit={(text) => handleEditSubmit(index, text)}
-                      onCancel={() => setEditInputId(undefined)}
-                      isSubmitDisabled={
-                        !conversationContext.canSubmitUserEntries
-                      }
-                    />
-                  )}
-                  {turn.selectedText && (
-                    <ActionTypeLabel actionType={turn.actionType} />
-                  )}
-                  {turn.selectedText && <Quote text={turn.selectedText} />}
-                  {showLongPageContentInfo && <LongPageInfo />}
+                        </>
+                        )}
+
+                        {showEditInput && (
+                          <EditInput
+                            text={firstEntryEdit.text}
+                            onSubmit={(text) => handleEditSubmit(index, text)} // TODO: uuid
```

Comment:
do we need to deal with this todo here?

Reply:
I changed modifyConversation to accept uuid